systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -142,6 +147,90 @@ void testAutoCreationOfCruiseControlTopicsWithResources(ExtensionContext extensi
         assertThat(partitionMetricsTopic.getReplicas(), is(2));
     }
 
+    @IsolatedTest
+    void testCreationOfCruiseControlConfigMap(ExtensionContext extensionContext) {
```

Comment:
What is the value of having a system test for this? The system tests should verify general functionality -> e.g. Cruise Control works as expected. To verify that ConfigMap is generated with the right content, you should use a unit test. You will need several milliseconds to test it instead of several minutes.

Reply:
You are right, removed